pyproject.toml

```diff
@@ -8,3 +8,38 @@ use_parentheses = true
 include_trailing_comma = true
 multi_line_output = 3
 force_grid_wrap = 0
+
+[tool.poetry]
+name = "geospatial-data-lake"
+version = "0.1.0"
+description = "Central storage, management and access for important geospatial datasets developed by LINZ"
+authors = [
+    "Bill Nelson <billgeo@users.noreply.github.com>",
```

Comment:
Incorrect Bill's email - should be `Bill M. Nelson <bmnelson@linz.govt.nz>`

Reply:
Thank you; changed. I got all the emails out of `git log --format='%aN <%aE>' | sort --unique`, so I wasn't sure whether I should "de-anonymize" Bill that way :)